GameServer/DBAgent.cpp

```diff
@@ -1806,7 +1806,7 @@ void CDBAgent::UpdateCandidacyNoticeBoard(string & strCharID, uint8 byNation, st
 
 	// Field is 1024 bytes in the database.
 	uint8 strBinaryNotice[1024] = {0};
-	uint16 sNoticeLen = strNotice.length();
+	uint16 sNoticeLen = (uint16)strNotice.length();
```

Comment:
```suggestion
	uint16 sNoticeLen = static_cast<uint16>(strNotice.length());
```
Please apply to all use cases where you used C cast-style to use C++ castin